### PR TITLE
`TCK`: Refactor - code duplication, helper functions

### PR DESCRIPTION
**Pull request by ivaylogarnev-limechain**
**Description**:
This PR refactors the code to reduce duplication and improve maintainability. It introduces several helper functions to streamline logic and enhance readability. Additionally, some constants were added to make the code more organized and easier to manage. These changes aim to improve both the clarity and scalability of the code.

**Related issue(s)**:
 #260 

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: test/token-service/test_tokenCreateTransaction.js
Comment (rwalworth) [inline test/token-service/test_tokenCreateTransaction.js:4726]: Another test skipped?

File: utils/helpers/constants/key-list.js
Comment (rwalworth) [inline utils/helpers/constants/key-list.js:55]: I think we can look to replace instances where `threeKeysKeyListParams` and `fivePrivateKeysKeyListParams` are used with this, since this covers all ed25519 and ecdsaSecp256k1 public and private keys

File: test/crypto-service/test_accountCreateTransaction.js
Comment (ivaylogarnev-limechain) [reply test/crypto-service/test_accountCreateTransaction.js:321]: t’s shorthand property syntax. When the key and variable have the same name, you can use this simplified format.

File: test/token-service/test_tokenCreateTransaction.js
Comment (ivaylogarnev-limechain) [reply test/token-service/test_tokenCreateTransaction.js:757]: All verification functions within the `update transaction` tests should be used with `retryOnError`, as there isn’t an issue with mirror node updates during creation.

File: utils/helpers/constants/key-list.js
Comment (ivaylogarnev-limechain) [reply utils/helpers/constants/key-list.js:55]: Yes, I think so too. The `fourKeysKeyListParams` includes all key types in use. I’ve updated it and tested it, and it seems to work perfectly!

File: test/crypto-service/test_accountCreateTransaction.js
Comment (ivaylogarnev-limechain) [reply test/crypto-service/test_accountCreateTransaction.js:991]: It looks like I missed adding `await` to this particular function when I updated others. Now that it’s added, it doesn’t seem to be working properly. I’ll add a description in the `notion file` so everyone can review it.

File: test/token-service/test_tokenCreateTransaction.js
Comment (ivaylogarnev-limechain) [reply test/token-service/test_tokenCreateTransaction.js:4726]: This is related to the TokenCreateTransaction custom fees bugs in services. I updated the Notion file— I thought I had added it earlier, but it seems I hadn’t. I’ve highlighted it in yellow; once you review it, you can unhighlight it.

File: test/token-service/test_tokenCreateTransaction.js
Comment (rwalworth) [reply test/token-service/test_tokenCreateTransaction.js:756]: Some functions were missing it, I pushed up adding it

File: test/token-service/test_tokenCreateTransaction.js
Comment (rwalworth) [reply test/token-service/test_tokenCreateTransaction.js:4725]: Ah it involves a deleted token, we can look at this one again on the `deleteToken` branch once this gets merged

File: test/crypto-service/test_accountCreateTransaction.js
Comment (rwalworth) [reply test/crypto-service/test_accountCreateTransaction.js:991]: I wasn't 100% sure where these would be saved in the response but I'd assume these are correct. I'll make the change and push up

**Review by rwalworth — APPROVED** [review]
LGTM, as long as JS sdk tests still pass this should be good to merge!

File: test/token-service/test_tokenCreateTransaction.js
Comment (ivaylogarnev-limechain) [reply test/token-service/test_tokenCreateTransaction.js:756]: Confirmed, all tests passed in JS.

**Pull request merged**
